week_01/code_wars/myeong/200828_GetMiddle.cpp

```diff
@@ -0,0 +1,15 @@
+std::string get_middle(std::string input) 
+{
+  int mid;
+  mid=input.length()/2;
```

Comment:
```suggestion
  int mid = input.length()/2;
```
내가 C++ 을 잘 몰라서 그러는데 혹시 위와 같이 쓰는 것도 되지 않나?? 만약 그렇다면, 한번에 적어주는게 더좋을 듯해~

Reply:
넵!